h/static/styles/annotations.scss

```diff
@@ -37,6 +37,26 @@
 .annotation-header { margin-top: 0 }
 .annotation-footer { margin-bottom: 0 }
 
+.annotation-section {
+  .excerpt { max-height: 4.8em; }
+  .excerpt-control a {
+    font-style: italic;
+    font-family: $serif-font-family;
+    font-weight: normal;
+  }
+  .excerpt--collapsed:after {
+    height: $base-line-height;
+    @include background(linear-gradient(to right,
+      rgba(255,255,255,0) 20%,
```

Comment:
Yes, its a bit nit-picky. The advantage is that we'll be able to add a lint to check for colors only being defined in one place in future.